### ISSUE TITLE
Freeze Tag: arena logic still present on koth maps

### ISSUE THREAD

**Issue by itVictor** [redsunservers/LoadoutBugTracker#49]
Back when koth maps were accidently added to the freeze tag rotation (and later kept), it seemed like a decent idea at the time, but now it seems like it's still doing some of the arena logic in king of the kill.

Specifically, the round will automatically end if all players on a team are dead/frozen, similarly to how arena works. However, this is pointless since in koth the players who are frozen will respawn after a time if they are not unfrozen by a teammate.

tl;dr fix this or remove koth from freeze tag

**Comment by ramunecarbonated** [comment]
The round should no longer end when all players are frozen on `koth_`, I am assuming this was unintended behaviour from a plugin update of some sort.
